src/test/java/com/amplitude/util/MockURLStreamHandler.java

```diff
@@ -0,0 +1,39 @@
+package com.amplitude.util;
+
+import java.io.IOException;
+import java.net.URL;
+import java.net.URLConnection;
+import java.net.URLStreamHandler;
+import java.net.URLStreamHandlerFactory;
+import java.util.HashMap;
+import java.util.Map;
+
+public class MockURLStreamHandler extends URLStreamHandler implements URLStreamHandlerFactory {
+
+  private final Map<URL, URLConnection> connections = new HashMap<>();
+
+  private final static MockURLStreamHandler instance = new MockURLStreamHandler();
+
+  public static MockURLStreamHandler getInstance(){
```

Comment:
nit: the space (){

Reply:
oh, let me run the formatter again on this.